policy/policy_file.go

```diff
@@ -657,6 +657,24 @@ func (c *Contents) GetPolicyName(org, filename string) string {
 	return ""
 }
 
+// check if another file in the same org has the same policy name as another file.
+// It returns the name of the conflict file.
+func (c *Contents) ConflictsWith(org, filename string, pol *Policy) string {
+	if !c.HasOrg(org) {
+		return ""
+	} else if c.HasFile(org, filename) {
```

Comment:
You dont need this check. The caller has already checked that policy file is not being tracked.

Reply:
There are 2 callers for PolicyFileChangeWatcher(), one is from the policy_manager and the other is from agreementbot. PolicyFileChangeWatcher uses this check to make sure the fileChanged function is not get called and this file is not put into the watcher map.